openwisp_users/api/serializers.py

```diff
@@ -32,6 +32,8 @@ class Meta:
             'created',
             'modified',
         )
+        # We need to remove the "UniqueValidator" class from the "slug" field
+        # to add our customised validation error message in the API response
```

Comment:
What about:

This allows replacing the default "UniqueValidator" for
the slug field with the custom validation defined below.

Reply:
done :+1: